fixtures/flows/brief/outcome.js

```diff
@@ -0,0 +1,144 @@
+export const create = async (params) => {
+    console.log('Starting to create outcome brief');
+    await selectLot('digital-outcome');
+    await createBrief();
+    await fillTitle(params.title);
+    await fillLocation(params.locations);
+    await fillDescriptinoOfWork();
```

Comment:
Typo in `Description` and in function definition.

Reply:
fixed